src/tests/ftest/deployment/agent_failure.yaml

```diff
@@ -57,14 +57,13 @@ ior:
     client_processes:
         np: 4
     iorflags:
-        flags: "-k -D 300 -v -w -W"
+        flags: -k -v -w -W
         api: DFS
         # This test doesn't work if the IOR runs too fast. In that case, increase
         # block_size and/or decrease transfer_size. To decrease transfer_size, divide it
         # by 2, 4, 8, etc.
-        transfer_size: '256K'
-        block_size: '20G'
-        write_x: 4
-        read_x: 2
-        dfs_oclass: "SX"
-        dfs_dir_oclass: "SX"
+        transfer_size: 256K
+        block_size: 20G
+        sw_deadline: 60
+        dfs_oclass: SX
+        dfs_dir_oclass: SX
```

Comment:
Just a cherry pick, so no need to change here, but FYI that `dfs_dir_oclass` is not used by IOR (only by mdtest)

Reply:
Okay. Thanks.